contracts/mocks/RefundableCrowdsaleImpl.sol

```diff
@@ -12,10 +12,11 @@ contract RefundableCrowdsaleImpl is RefundableCrowdsale {
     uint256 _rate,
     address _wallet,
     uint256 _goal,
+    RefundVault _vault,
     MintableToken _token
```

Comment:
Let's use the same params order (`vault, token` or `token, vault`) for both the mock here and the SampleCrowdsale example, just for the sake of consistency.

Reply:
@spalladino 100% agree. Just pushed a fix